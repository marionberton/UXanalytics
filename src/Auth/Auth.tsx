import React from 'react';

export const Auth = () => (
  <section className="grid lg:grid-cols-3">
    <div className="lg:col-span-1 lg:order-1 order-3">
      <div className="lg:h-screen lg:w-full flex flex-col justify-center items-center">
        <div className="items-center lg:w-3/4 md:w-2/4 text-left lg:m-0 lg:p-0 my-8 sm:mx-0">
          <h1 className="xl:text-5xl md:text-4xl sm:text-xl text-darkgrey font-title font-semibold">
            Hello!
          </h1>
          <p className="xl:text-2xl md:text-xl sm:text-lg  text-darkgrey font-title font-medium ">
            Create your free account
          </p>
          <p className="text-base text-darkgrey xl:font-normal lg:font-light mt-4">
            Try our demo and be one of the first to use UX Analytic, and help
            the service grow! We'll email you our beta release when it's ready.
          </p>
          <p className="text-base text-darkgrey xl:font-normal lg:font-light mt-4">
            <span className="text-purple-700 font-medium">Note:</span>This
            version of UX Analytic is an MVP, and is currently still in
            developement..
          </p>
          <button
            className="bg-btnred text-white font-title font-semibold py-2 px-4 rounded-full mt-10 w-1/3"
            type="button"
          >
            signin
          </button>
        </div>
      </div>
    </div>

    <div className="backgroundShape bg-lightgrey lg:col-span-2 lg:order-3 order-1">
      <div className="lg:h-screen lg:w-full flex justify-center items-center h-64">
        <h1 className="xl:text-5xl lg:text-4xl md:text-2xl text-darkgrey font-title font-semibold lg:w-2/3 md:w-3/5 text-left leading-none px-8">
          The goal is to turn data into information & information into insight
          <br />
          <span className="font-normal text-2xl">â€” Carly Fiorina</span>
        </h1>
      </div>
    </div>
  </section>
);
