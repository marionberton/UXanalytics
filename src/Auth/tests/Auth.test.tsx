import React from 'react';
import { mount } from 'enzyme';

import { Auth } from '../Auth';

describe('<Auth/>', () => {
  it('renders the component', () => {
    const component = mount(<Auth />);

    expect(component.render()).toMatchSnapshot();
  });
});
