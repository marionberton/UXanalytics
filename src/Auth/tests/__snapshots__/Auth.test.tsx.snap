// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Auth/> renders the component 1`] = `
<section
  class="grid lg:grid-cols-3"
>
  <div
    class="lg:col-span-1 lg:order-1 order-3"
  >
    <div
      class="lg:h-screen lg:w-full flex flex-col justify-center items-center"
    >
      <div
        class="items-center lg:w-3/4 md:w-2/4 text-left lg:m-0 lg:p-0 my-8 sm:mx-0"
      >
        <h1
          class="xl:text-5xl md:text-4xl sm:text-xl text-darkgrey font-title font-semibold"
        >
          Hello!
        </h1>
        <p
          class="xl:text-2xl md:text-xl sm:text-lg  text-darkgrey font-title font-medium "
        >
          Create your free account
        </p>
        <p
          class="text-base text-darkgrey xl:font-normal lg:font-light mt-4"
        >
          Try our demo and be one of the first to use UX Analytic, and help the service grow! We'll email you our beta release when it's ready.
        </p>
        <p
          class="text-base text-darkgrey xl:font-normal lg:font-light mt-4"
        >
          <span
            class="text-purple-700 font-medium"
          >
            Note:
          </span>
          This version of UX Analytic is an MVP, and is currently still in developement..
        </p>
        <button
          class="bg-btnred text-white font-title font-semibold py-2 px-4 rounded-full mt-10 w-1/3"
          type="button"
        >
          signin
        </button>
      </div>
    </div>
  </div>
  <div
    class="backgroundShape bg-lightgrey lg:col-span-2 lg:order-3 order-1"
  >
    <div
      class="lg:h-screen lg:w-full flex justify-center items-center h-64"
    >
      <h1
        class="xl:text-5xl lg:text-4xl md:text-2xl text-darkgrey font-title font-semibold lg:w-2/3 md:w-3/5 text-left leading-none px-8"
      >
        The goal is to turn data into information & information into insight
        <br />
        <span
          class="font-normal text-2xl"
        >
          â€” Carly Fiorina
        </span>
      </h1>
    </div>
  </div>
</section>
`;
